cosmwasm/deploy-contract.js

```diff
@@ -29,31 +29,28 @@ const upload = (client, wallet, chainName, config, options) => {
     if (!fetchCodeId && (!reuseCodeId || isNil(contractConfig.codeId))) {
```

Comment:
codeId should be extracted here. And then `contractConfig` should be defined as either the top-level config or chain specific (and initialize the chain specific config to {} if it doesn't exist), so you don't have the `if (chainConfig)` checks everywhere

Reply:
refactoring contractConfig will be done in a different PR after evaluating the best way to also move codeId and other top level contract configs to the contract chain config